crates/hyperswitch_domain_models/src/errors/api_error_response.rs

```diff
@@ -290,6 +290,8 @@ pub enum ApiErrorResponse {
     InvalidPlatformOperation,
     #[error(error_type = ErrorType::InvalidRequestError, code = "IR_45", message = "External vault failed during processing with connector")]
     ExternalVaultFailed,
+    #[error(error_type = ErrorType::InvalidRequestError, code = "IR_46", message = "Field {fields} doesn't match with the ones used during mandate creation")]
```

Comment:
```suggestion
    #[error(error_type = ErrorType::InvalidRequestError, code = "IR_46", message = "Fields {fields} doesn't match with the ones used during mandate creation")]
```

Shouldn't it be fields only? as you are joining multiple fields in it